db/migrate/20160510180328_add_small_biz_flag_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSmallBizFlagToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :small_business, :boolean, default: false, null: false
```

Comment:
let's index on this too, since we're going to be querying on it. See section on "partial indexes" here http://rny.io/rails/postgresql/2013/08/20/postgresql-indexing-in-rails.html